From 197426b349edef0882efd7bab93b8c5fd9e7da22 Mon Sep 17 00:00:00 2001
From: Mathieu Lirzin <mthl@gnu.org>
Date: Tue, 16 May 2017 15:59:12 +0200
Subject: [PATCH] automake: Update 'read_am_file' docstring.

* bin/automake.in (read_am_file): Update docstring which was referring
to a non existent '%contents' variable.
---
 bin/automake.in | 11 +++++++----
 1 file changed, 7 insertions(+), 4 deletions(-)

diff --git a/bin/automake.in b/bin/automake.in
index 221d7d3..d356336 100644
--- a/bin/automake.in
+++ b/bin/automake.in
@@ -6208,10 +6208,13 @@ sub check_trailing_slash ($\$)
 
 # read_am_file ($AMFILE, $WHERE, $RELDIR)
 # ---------------------------------------
-# Read Makefile.am and set up %contents.  Simultaneously copy lines
-# from Makefile.am into $output_trailer, or define variables as
-# appropriate.  NOTE we put rules in the trailer section.  We want
-# user rules to come after our generated stuff.
+# Read $AMFILE file name which is located in $RELDIR, and set up
+# global variables resetted by '&generate_makefile'.  Simultaneously
+# copy lines from $AMFILE into '$output_trailer', or define variables
+# as appropriate.
+#
+# NOTE: We put rules in the trailer section.  We want user rules to
+# come after our generated stuff.
 sub read_am_file
 {
     my ($amfile, $where, $reldir) = @_;
-- 
1.9.1

